communication/moderators.md

```diff
@@ -142,32 +142,26 @@ Email: contributors@kubernetes.io
 
 ### Primary Moderators
 
-Moderators seats: 12
+Moderators seats: 6
 
 | Name                | Kubernetes Slack ID   | Region   | Timezone                                                |
 | ------------------- | --------------------- | -------- | ------------------------------------------------------- |
-| Alison Dowdney      | @alisondy             | EMEA     | [GMT - Greenwich Mean Time](https://time.is/GMT)        |
-| Jeffrey Sica        | @jeefy                | Americas | [ET - Eastern Time (US East Coast)](https://time.is/ET) |
-| Bob Killen          | @mrbobbytables        | Americas | [ET - Eastern Time (US East Coast)](https://time.is/ET) |
 | Chris Short         | @chrisshort           | Americas | [ET - Eastern Time (US East Coast)](https://time.is/ET) |
-| Taylor Dolezal      | @onlydole             | Americas | [PT - Pacific Time (US West Coast)](https://time.is/PT) |
-| David McKay         | @rawkode              | EMEA     | [GMT - Greenwich Mean Time](https://time.is/GMT)        |
-| Marky Jackson       | @markyjackson-taulia  | Americas | [PT - Pacific Time (US West Coast)](https://time.is/PT) |
 | Nigel Brown         | @nigel                | Americas | [CT - Central Time (US Central)](https://time.is/CT)    |
```

Comment:
Cannot find nigel on Slack:

<img width="424" alt="image" src="https://github.com/kubernetes/community/assets/9950081/b9fe4696-85e5-41ce-aa64-6d1cd63f4c8f">

Reply:
oh, it says there are multiple options

Reply:
sorry for the noise. Figured it out